devices.js

```diff
@@ -8216,7 +8216,7 @@ const devices = [
         model: 'IM6001-MPP01',
         vendor: 'SmartThings',
         description: 'Multipurpose sensor (2018 model)',
-        fromZigbee: [fz.temperature, fz.ias_contact_alarm_1, fz.battery, fz.smartthings_acceleration],
+        fromZigbee: [fz.temperature, fz.battery, fz.ias_contact_alarm_1, fz.smartthings_acceleration],
```

Comment:
is this change necessary? I was able to get accelerometer data to report without changing this line at all.

Reply:
Naaah, It just happened when I tested and looked at the one above (mutliv4) and used the same.